src/safeds/data/tabular/containers/_table.py

```diff
@@ -695,6 +697,34 @@ def get_row(self, index: int) -> Row:
 
         return Row._from_pandas_dataframe(self._data.iloc[[index]], self._schema)
 
+    def _get_similar_columns(self, column_name: str) -> list[str]:
+        """
+        Get all similar column names to the given one that's not in the table.
```

Comment:
Since we also return names that match the given name exactly:

```suggestion
        Get all the column names in a Table that are similar to a given name.
```